src/mpid/ch4/generic/am/mpidig_am_init.c

```diff
@@ -56,7 +56,8 @@ int MPIDIG_am_init(void)
     MPL_atomic_store_uint64(&MPIDI_global.exp_seq_no, 0);
     MPL_atomic_store_uint64(&MPIDI_global.nxt_seq_no, 0);
 
-    MPIDI_global.buf_pool = MPIDIU_create_buf_pool(MPIDIU_BUF_POOL_NUM, MPIDIU_BUF_POOL_SZ);
+    MPIDI_global.buf_pool =
+        MPIDIU_create_buf_pool(MPIDIU_BUF_POOL_NUM, MPIDIU_BUF_POOL_SZ, MPIDIU_BUF_POOL_PAGEABLE);
```

Comment:
Leave a comment here: 

We made an agreement on last online meeting that, generic GPU is serving as an utility layer for two parts: data movement and pipeline structure. 

Every layer, for example `netmod/ofi` or `ch4/generic/am` layer, if it misses internal capability for handling GPU, it should call utility function from GPU utility layer.

To follow that design, buffer pool registration should happen inside GPU utility layer, not here in AM.